packages/rpc-subscriptions/src/__tests__/rpc-subscriptions-functional-test.ts

```diff
@@ -0,0 +1,40 @@
+import { Address } from '@solana/addresses';
+import { AccountNotificationsApi, SolanaRpcSubscriptionsApi } from '@solana/rpc-subscriptions-api';
+import { createSubscriptionRpc, RpcSubscriptions } from '@solana/rpc-subscriptions-spec';
+
+import {
+    createDefaultRpcSubscriptionsTransport,
+    createDefaultSolanaRpcSubscriptionsChannelCreator,
+    createSolanaRpcSubscriptionsApi,
+} from '..';
+
+function createLocalhostSolanaRpcSubscriptions(): RpcSubscriptions<SolanaRpcSubscriptionsApi> {
+    return createSubscriptionRpc({
+        api: createSolanaRpcSubscriptionsApi(),
+        transport: createDefaultRpcSubscriptionsTransport({
+            createChannel: createDefaultSolanaRpcSubscriptionsChannelCreator({ url: 'ws://localhost:8900' }),
+        }),
+    });
+}
+
+describe('accountNotifications', () => {
+    let rpcSubscriptions: RpcSubscriptions<AccountNotificationsApi>;
+    beforeEach(() => {
+        rpcSubscriptions = createLocalhostSolanaRpcSubscriptions();
+    });
+
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('can subscribe to account notifications', async () => {
```

Comment:
I think you can actually write this as `it.failing` instead of skipping it.

Reply:
I tried that, but because it fails by timing out it doesn't seem to work.